### PR TITLE
[13.0][MIG] stock_move_location

### PR DESCRIPTION
**Pull request by JoanSForgeFlow**
stock_move_location migration to v13.

cc ~ @ForgeFlow @LoisRForgeFlow @AdriaGForgeFlow

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_move_location/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Move Stock Location",
-    "version": "12.0.1.2.0",
+    "version": "13.0.1.0.0",
     "author": "Julius Network Solutions, " "Odoo Community Association (OCA)",
```
Comment (AdriaGForgeFlow) [inline stock_move_location/__manifest__.py]: Could you remove extra double quotes?

**Review by AdriaGForgeFlow — APPROVED** [review]
Code looks good to me, just a comment below.

And, please, check Travis (test is failing probably due to another module)

File: stock_move_location/tests/test_common.py [hunk 2]
```diff
@@ -14,56 +14,67 @@ def setUpClass(cls):
...
         cls.quant_obj = cls.env["stock.quant"]
+        cls.company = cls.env.ref("base.main_company")
 
         # Enable multi-locations:
-        wizard = cls.env["res.config.settings"].create(
-            {"group_stock_multi_locations": True}
-        )
-        wizard.execute()
+        group = cls.env.ref("stock.group_stock_multi_locations")
+        cls.env.user.groups_id = [(4, group.id, 0)]
```
Comment (LoisRForgeFlow) [inline stock_move_location/tests/test_common.py]: You can remove this, as it is now done in the init hook

File: stock_move_location/tests/test_move_location.py [hunk 3]
```diff
@@ -13,7 +13,8 @@ def setUp(self):
         self.setup_product_amounts()
 
     def _create_wizard(self, origin_location, destination_location):
-        return self.wizard_obj.create(
+        MoveLocationWizard = self.env["wiz.stock.move.location"]
```
Comment (LoisRForgeFlow) [inline stock_move_location/tests/test_move_location.py]: this should go in the setup of the test and in lowercase.

File: stock_move_location/wizard/stock_move_location.py [hunk 4]
```diff
@@ -29,6 +28,7 @@ def _get_default_picking_type_id(self):
     origin_location_disable = fields.Boolean(
         compute="_compute_readonly_locations",
         help="technical field to disable the edition of origin location.",
+        default=False,
```
Comment (LoisRForgeFlow) [inline stock_move_location/wizard/stock_move_location.py]: is there a need for this now? in previous versions, Booleans were False by default.

File: stock_move_location/wizard/stock_move_location.py [hunk 5]
```diff
@@ -39,6 +39,7 @@ def _get_default_picking_type_id(self):
     destination_location_disable = fields.Boolean(
         compute="_compute_readonly_locations",
         help="technical field to disable the edition of destination location.",
+        default=False,
```
Comment (LoisRForgeFlow) [inline stock_move_location/wizard/stock_move_location.py]: same as above

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
See some comments inline. Please squash migration commits into one.

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by LoisRForgeFlow — APPROVED** [review]
thanks!

**Pull request merged**